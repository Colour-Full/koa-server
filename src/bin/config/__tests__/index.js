import { getSpaEnvVars } from '..'

describe('configMiddleware', () => {
  describe('getSpaEnvVars', () => {
    const env = {
      PORT: '9001',
      LOGSTASH_MODE: 'console',
      SPA_LOGSTASH_MODE: 'logstash',
      SPA_LOGSTASH_HOST: 'logstash.rullion.co.uk'
    }

    it('works', () => {
      const result = getSpaEnvVars(env)
      expect(result).toBeTruthy()
      expect(result).toMatchSnapshot()
    })
  })
})
