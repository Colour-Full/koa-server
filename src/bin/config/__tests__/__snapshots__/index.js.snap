// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configMiddleware getSpaEnvVars works 1`] = `
Object {
  "LOGSTASH_HOST": "logstash.rullion.co.uk",
  "LOGSTASH_MODE": "logstash",
}
`;
